components/modal.jsx

```diff
@@ -0,0 +1,31 @@
+// [THIS LINE TO BE REMOVED] bootstrap class reference: http://nakupanda.github.io/bootstrap3-dialog/
+
+var React = require('react');
+var ReactAddons = require('react/addons');
+
+var Modal = React.createClass({
+  mixins: [ReactAddons.PureRenderMixin],
+  handleClick: function(e) {
+    if (e.target === this.getDOMNode().querySelector("button.close")) {
+      this.props.onClose();
+    }
+  },
+  render: function() {
+    return (
+      <div className="modal show">
+        <div className="modal-backdrop"></div>
+        <div className="modal-dialog" onClick={this.handleClick}>
+          <div className="modal-header">
+            <button type="button" className="close" data-dismiss="modal" aria-hidden="true">x</button>
```

Comment:
No worries, @toolness .  I don't think that attribute is ever used in my code.  I should've excluded it!  Thanks for spotting this